### ISSUE TITLE
"run_lissero somestring" gives vomit backtrace

### ISSUE THREAD

**Issue by tseemann** [MDU-PHL/LisSero#11]
```
linuxbrew@deepthought:~ $ run_lissero  foo
Traceback (most recent call last):
  File "/home/linuxbrew/.linuxbrew/bin/run_lissero", line 8, in <module>
    sys.exit(run_lissero())
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/lissero/run_lissero.py", line 52, in run_lissero
    path_serodb = os.path.realpath(serotype_db)
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/posixpath.py", line 394, in realpath
    filename = os.fspath(filename)
TypeError: expected str, bytes or os.PathLike object, not NoneType

```

**Comment by abcdtree** [comment]
It looks like both #11 and #12 are bugs which are not about input file. 
```File "/home/linuxbrew/.linuxbrew/opt/python/lib/python3.7/site-packages/lissero/run_lissero.py", line 52, in run_lissero
    path_serodb = os.path.realpath(serotype_db)``` It's about the serotype_db which should be pre-set if no serotype_db paremeter was provided.

**Comment by andersgs** [comment]
Fixed in latest release (0.4.1)
